tests/job-spawner-and-cleaner.py

```diff
@@ -4,22 +4,28 @@
 
 import redis
 
+
 sys.path.append('../python')
 from redis_work_queue import KeyPrefix, Item, WorkQueue
 
+queue_list = sys.argv[2].split(" ")
+print(queue_list)
+
 if len(sys.argv) < 2:
     raise Exception("first command line argument must be redis host")
-db = redis.Redis(host=sys.argv[1])
+db = redis.Redis(host=sys.argv[1].replace(":6379",""))
 
 if len(db.keys("*")) > 0:
     raise Exception("redis database isn't clean")
 
-python_queue = WorkQueue(KeyPrefix("python_jobs"))
-rust_queue = WorkQueue(KeyPrefix("rust_jobs"))
-go_queue = WorkQueue(KeyPrefix("go_jobs"))
-typeScript_queue = WorkQueue(KeyPrefix("typeScript_jobs"))
-#dotnet_queue = WorkQueue(KeyPrefix("dotnet_jobs"))
-shared_queue = WorkQueue(KeyPrefix("shared_jobs"))
+
+queue_list.append("shared_jobs")
+queue_list_names = []
+
+for index in range(len(queue_list)):
+    queue_list_names.append(queue_list[index])
+    queue_list[index]=WorkQueue(KeyPrefix(queue_list[index]))
+
```

Comment:
We could have used `.insert(0, item)` to prepend the items.

However, there's no need for that. I think it's much cleaner and way more readable to keep `shared_queue` as a separate variable and only put the language specific queues in the list.